test/services/serviceProvider.test.ts

```diff
@@ -26,7 +109,7 @@ describe('ServiceProvider', () => {
     );
 
     const service = await ServiceProvider.getService(
-      ServiceType.Logger,
+      ServiceType.Telemetry,
```

Comment:
I see that all instances of Logger in this test file are replaced with Telemetry.  Are we no longer going to test Logger?